innovation.game.php

```diff
@@ -18,13 +18,18 @@
 
 
 require_once(APP_GAMEMODULE_PATH.'module/table/table.game.php');
+require_once('modules/Innovation/Utils/Arrays.php');
 require_once('modules/Innovation/Utils/Strings.php');
+require_once('modules/Innovation/GameState.php');
 
 /* Exception to be called when the game must end */
 class EndOfGame extends Exception {}
 
 class Innovation extends Table
 {
+    /** @var \Innovation\GameState An inverted control structure for accessing game state in a testable manner */
+    private \Innovation\GameState $innState;
```

Comment:
I'm not sold on the naming of this variable. Is there a particular reason you didn't use `gameState`?

Reply:
Yes, since this extends an internal BGA class - `Table` - I wanted guard against them potentially changing things internally, so prefixing with an Innovation-specific name. There's a _very_ good chance they already have a `$gameState` instance variable on the class this would break.

Reply:
Would something like `innGameState` be preferred? I'm hesitant to add any vanilla instance variables here without a prefix.

Reply:
Makes sense. How about "innovationGameState"? When I read "inn", it doesn't translate to "innovation" in my head